posthog/settings/web.py

```diff
@@ -25,6 +25,10 @@
 DECIDE_BUCKET_CAPACITY = get_from_env("DECIDE_BUCKET_CAPACITY", type_cast=int, default=500)
 DECIDE_BUCKET_REPLENISH_RATE = get_from_env("DECIDE_BUCKET_REPLENISH_RATE", type_cast=float, default=10.0)
 
+# Decide db settings
+
+DECIDE_SKIP_DATABASE_FLAGS = get_from_env("DECIDE_SKIP_DATABASE_FLAGS", False, type_cast=str_to_bool)
```

Comment:
Is redis not a database?

Reply:
Feels like this is better named `DECIDE_SKIP_POSTGRES_FLAGS`

Reply:
Only a cache to me 😋 , but yeah reasonable, updated 👍